launch/sensing_launch/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>sensing_launch</name>
+  <version>0.1.0</version>
+  <description>The sensing_launch package</description>
+
+  <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <exec_depend>aip_x1_launch</exec_depend>
+  <exec_depend>aip_x2_launch</exec_depend>
+  <exec_depend>aip_xx1_launch</exec_depend>
```

Comment:
@1222-takeshi 
aip_xx1_launch uses `velodyne_point_cloud` pkg so you need to add build_depends.repos

Reply:
Here is a repo https://github.com/tier4/velodyne_vls

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/165/commits/0708b9fc8da727f943849dfc7877f6b4f993fc90